packages/material-ui-lab/src/Alert/Alert.d.ts

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { StandardProps } from '@material-ui/core';
+import { PaperProps } from '@material-ui/core/Paper';
+
+export type Color = 'success' | 'info' | 'warning' | 'error';
+
+export interface AlertProps extends StandardProps<PaperProps, AlertClassKey, 'variant'> {
+  /**
+   * The action to display.
```

Comment:
It may be worth clarifying that this (from a DOM perspective) in the position of the close button.  I'm not sure how to word that though.  No biggie either way.

Reply:
It's copied and paste from `SnackbarContent.actions`'s description. Happy to improve both at the same time. Maybe?

```
The action to display. It renders after the message, at the end of the alert.
```

Reply:
sounds good to me.

Reply:
updated in https://github.com/mui-org/material-ui/pull/18702/commits/ffd00685bda70b66868f6f911be8683235c1b132